RELEASE-NOTES.md

```diff
@@ -113,8 +113,8 @@ _Read on if you're a developer. Or curious. Or both._
     - `pm.GaussianRandomWalk` initial distribution defaults to zero-centered normal with sigma=100 instead of flat (see[#5779](https://github.com/pymc-devs/pymc/pull/5779))
     - `pm.AR` initial distribution defaults to unit normal instead of flat (see[#5779](https://github.com/pymc-devs/pymc/pull/5779))
 
-  - `logpt`, `logpt_sum`, `logp_elemwiset` and `nojac` variations were removed. Use `Model.logpt(jacobian=True/False, sum=True/False)` instead.
-  - `dlogp_nojact` and `d2logp_nojact` were removed. Use `Model.dlogpt` and `d2logpt` with `jacobian=False` instead.
+  - `logpt`, `logpt_sum`, `logp_elemwiset` and `nojac` variations were removed. Use `Model.logp(jacobian=True/False, sum=True/False)` instead.
+  - `dlogp_nojact` and `d2logp_nojact` were removed. Use `Model.dlogp` and `d2logp` with `jacobian=False` instead.
```

Comment:
We shouldn't change the old Release Notes as this refer to the already released V4.0.0. Instead you should add a new release note in the most recent section explaining the changes from `t` suffix to no suffix.

Reply:
This weekend I already created a `4.0.1` section in the release notes.
You'll have to rebase, otherwise there will be git conflicts..